src/visitors/cssNamespace.js

```diff
@@ -21,7 +21,7 @@ export default (path, state) => {
   const cssNamespace = getCssNamespace(state);
 
   if (
-    (isStyled(path.node.tag, state) || isHelper(path.node.tag, state)) &&
+    isStyled(path.node.tag, state) &&
```

Comment:
Minor change: Can you also remove `isHelper` from the imports from the top as it is no longer used?

It looks like we are missing the eslint rule for that. I can add that in a follow up PR so we don't have to catch these manually.

Reply:
yup I'll do that now